src/Moq/ExpressionExtensions.cs

```diff
@@ -169,4 +169,25 @@ internal static (Expression Object, MethodInfo Method, IReadOnlyList<Expression>
 				expression.ToStringFixed()));
 		}
 	}
+
+	internal readonly struct CallInfo
+	{
+		private readonly Expression expression;
+		private readonly MethodInfo method;
+		private readonly IReadOnlyList<Expression> arguments;
+
+		public CallInfo(Expression expression, MethodInfo method, IReadOnlyList<Expression> arguments)
+		{
+			this.expression = expression;
+			this.method = method;
+			this.arguments = arguments;
+		}
+
+		public void Deconstruct(out Expression expression, out MethodInfo method, out IReadOnlyList<Expression> arguments)
```

Comment:
Using a `Deconstruct` instead of just making the fields `public` means that we can still use tuple syntax / deconstruction in assignments.